frontend/pages/audit-logs.vue

```diff
@@ -105,6 +194,11 @@ const allColumns: Column[] = [
     field: 'action',
     header: tt('Action'),
     customBody: true,
+    filterType: FilterType.EnumBased,
+    enumValues: Object.values(AuditLogAction).map((a: AuditLogAction) => ({
+      label: `${a}`.replace('AuditLogAction', ''),
```

Comment:
You don't need to stringify the enum, typescript is smart enough to know it's a string enum:

```suggestion
      label: a.replace('AuditLogAction', ''),
```

That said, this is fragile in the sense that it depends on some internal behavior of our TS codegen. I think it's fine in this case since its just a convenience thing, but the more robust (and tedious) approach would be to maintain the mapping explicitly